### PR TITLE
[DOCS]chore: import external repo readmes

### PR DESCRIPTION
**Pull request by sstraatemans**

### CODE REVIEW

File: packages/apps/docs/src/scripts/remarkFrontmatterToProps.mjs [hunk 1]
```diff
@@ -23,7 +23,7 @@ const getCurrentPostFromJson = (root) => {
   const data = getFlatData();
 
   return data.find((item) => {
-    return item && item.root === root;
+    return item && (item.root === root || `${item.root}/` === root);
```
Comment (sstraatemans) [inline packages/apps/docs/src/scripts/remarkFrontmatterToProps.mjs:26]: fixes the fact that some pages did not show a lastmodifieddate

File: packages/apps/docs/src/scripts/build.mjs [hunk 2]
```diff
@@ -73,6 +74,7 @@ const initFunc = async (fnc, description) => {
 };
 
 (async function () {
+  deleteTempDir();
```
Comment (sstraatemans) [inline packages/apps/docs/src/scripts/build.mjs:77]: starting with a clean slate

File: packages/apps/docs/src/pages/build/quickstart/index.md [hunk 3]
```diff
@@ -1,356 +0,0 @@
----
-title: 10 Minute Quickstart
-description: Learn Kadena’s core concepts & tools for development in 10 minutes
-menu: Quickstart
-label: 10 Minute Quickstart
-order: 1
-layout: full
-tags: [devnet, chainweaver, tutorial, docker, transactions]
----
```
Comment (realdreamer) [inline packages/apps/docs/src/pages/build/quickstart/index.md:9]: what's the reason for deleting this file?

Comment (realdreamer) [reply packages/apps/docs/src/pages/build/quickstart/index.md:9]: okay got it! we're getting this file from the getting started read ma page 👍🏽

**Review by realdreamer — APPROVED** [review]

**Pull request merged**